js/logging.js

```diff
@@ -69,7 +69,12 @@ if (window.console) {
 // The mechanics of preparing a log for publish
 
 function getHeader() {
-  return window.navigator.userAgent + ' node/' + window.config.node_version;
+  let header = window.navigator.userAgent;
+
+  header += ' node/' + window.config.node_version;
+  header += ' env/' + window.config.environment;
+
+  return header;
```

Comment:
Isn't the node version in there already? I'm seeing something like "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Signal/1.0.34 Chrome/56.0.2924.87 Electron/1.6.14 Safari/537.36 **node/7.4.0**" before this change.

Reply:
Yep that (line 74) was just a formatting change to make the second append I added (line 75) look the same as the first.

Reply:
Oh I see now, my bad.